pages/components/docs/Flex.md

```diff
@@ -1,41 +1,36 @@
 # Flex and Flex.Item
```

Comment:
Should we remove `Flex.Item` from this heading?
```suggestion
# Flex
```

Reply:
oh yis i forgot to push up my commit that did this lol